addons/stock/models/stock_move_line.py

```diff
@@ -895,6 +895,21 @@ def action_open_reference(self):
             'res_id': self.id,
         }
 
+    def action_open_picking(self):
+        self.ensure_one()
+        action = {
+            'type': 'ir.actions.act_window',
+            'views': [[False, "form"]]
+        }
+        if self.picking_id:
+            action['res_model'] = self.picking_id._name
+            action['res_id'] = self.picking_id.id
+        else:
+            action['res_model'] = self._name
+            action['res_id'] = self.id
+        return action
```

Comment:
maybe you can re-use `action_open_reference()`?

Reply:
Yes, indeed. It does the same job.